etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include npm.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}/wayland-*
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.npm
+noblacklist ${HOME}/.npmrc
+
+#include allow-nodejs.inc
+noblacklist ${PATH}/node
+noblacklist /usr/include/node
```

Comment:
```suggestion
```

There is no `include disable-interpreters.inc` where the only blacklist of this files happen. (And if it would be using the `allow-*.inc` files is desired to simplify maintenance.)